guide/app_star_catalogue.tex

```diff
@@ -98,7 +98,7 @@ \subsection{General Description}%\label{general-description}
 \midrule
 \file{stars\_0\_0v0\_8.cat} & 0 & 28 bytes & 0 &      4,979 & Hipparcos\\
 \file{stars\_1\_0v0\_8.cat} & 0 & 28 bytes & 1 &     21,806 & Hipparcos\\
-\file{stars\_2\_0v0\_8.cat} & 0 & 28 bytes & 2 &    150,824 & Hipparcos\\
+\file{stars\_2\_0v0\_8.cat} & 0 & 28 bytes & 2 &    150,826 & Hipparcos\\
```

Comment:
How did you get this value? As I was reading the documentation on the catalog header, I noticed that the magic value given there (`0xde0955a3`) is not what's actually present in the file (`0x835f040a`), and what's defined in `ZoneArray.hpp` in `FILE_MAGIC` (same `0x835f040a`). So I'm not sure if the whole doc there is up to date.

Reply:
I got the value by parsing the existing catalog. The magic value `0x0a045f83` works for me when I generate new catalog (and everything inside is encoded in little endian). Do you want me to change `0xde0955a3` to `0x0a045f83`?
<img width="726" alt="image" src="https://github.com/user-attachments/assets/b12cf886-3a0e-486a-ab4f-79a0b57c69e2">

Reply:
Uh you are right, I understand what you mean now. I have updated the guide, other description seems fine to me as far as helping me to parse the existing catalog myself.

Reply:
The star catalog and docs are indeed old. (2006?) The possibly outdated docs on it were collected from the old wiki which was often left behind when something changed. I agree, nowadays the native format should be little-endian, with byte-swapping on the rare big-endians. There is still the proper motion issue to be handled (Guide, F.2 and #3110), I hope @henrysky has this in mind for the next step.